platform-service-framework/src/main/java/org/hypertrace/core/serviceframework/PlatformService.java

```diff
@@ -108,6 +110,9 @@ public void start() {
     context.addServlet(new ServletHolder(new MetricsServlet()), "/metrics");
     context.addServlet(new ServletHolder(new HealthCheckServlet(this)), "/health");
     context.addServlet(new ServletHolder(new ThreadDumpServlet()), "/threads");
+    context.addServlet(new ServletHolder(new CpuProfileServlet()), "/pprof");
```

Comment:
Can we add one for getting Histogram too?

Reply:
@avinashkolluru : This is already there as part of the PR.

Reply:
Ahhh nice. Thanks